### ISSUE TITLE
Update to API Umbrella 0.14

### ISSUE THREAD

**Issue by bajiat** [apinf/platform#2176]
NREL has made 0.14 release of API Umbrella. See [release notes](https://github.com/NREL/api-umbrella/releases/tag/v0.14.0).

The 0.14.0 release has fixed some issues requested by APInf team, e.g. [Filtering analytics data by API backend](https://github.com/NREL/api-umbrella/issues/252)

**Comment by bajiat** [comment]
@shaliko Could you cooperate with @brylie on this?

**Comment by shaliko** [comment]
@bajiat Yes, and @brylie already pinged me about API Umbrella 0.14. I checked release notes, it will require update out docker-compose config and them do some tests as an example on nightly. 

When I can start work on it?

**Comment by bajiat** [comment]
@shaliko What if we have only a 0.14 deployment (one new proxy for nightly)?

**Comment by shaliko** [comment]
No problem, I will be free for that task on next week is it OK?

**Comment by bajiat** [comment]
@shaliko That is ok.

**Comment by shaliko** [comment]
@bajiat @saralavanip Deploy on nightly done.
